test/unit/auth/auth-config.spec.ts

```diff
@@ -206,10 +206,26 @@ describe('MultiFactorAuthConfig', () => {
       const config = new MultiFactorAuthConfig({
         state: 'ENABLED',
         enabledProviders: ['PHONE_SMS'],
+        providerConfigs: [
```

Comment:
Do you have any tests on these field for the validation you wrote, they could be either located in auth-config.spec or auth.spec